L1Trigger/CSCTriggerPrimitives/src/CSCTriggerPrimitivesBuilder.cc

```diff
@@ -621,6 +621,12 @@ void CSCTriggerPrimitivesBuilder::build(const CSCBadChambers* badChambers,
   // run MPC simulation
   m_muonportcard->loadDigis(oc_lct);
 
+  // temporary hack to ensure that all MPC LCTs are read out
```

Comment:
@fwyzard I tried adjusting the python configuration https://github.com/cms-sw/cmssw/blob/CMSSW_8_1_X/L1Trigger/CSCCommonTrigger/python/CSCCommonTrigger_cfi.py but that one is also picked up by csctfDigis. A shift of the BX readout window there does not make the agreement better. I'll try to figure out with CSC DPG how we can make the code more transparent for MC/DATA.